scipy/stats/_hypotests.py

```diff
@@ -1433,3 +1435,242 @@ def cramervonmises_2samp(x, y, method='auto'):
             p = max(0, 1. - _cdf_cvm_inf(tn))
 
     return CramerVonMisesResult(statistic=t, pvalue=p)
+
+
+class TukeyKramerResult:
+    """Result of `scipy.stats.tukeykramer`.
+
+    Attributes
+    ----------
+    sig_level : float
+            The desired significance level (copied from `tukeykramer` input).
+    statistic : float ndarray
+        The computed statistic of the test for each comparison. The
+        ``(i, j)`` index is the p-value for the ith - jth group comparison.
+    pvalue : float ndarray
+        The associated p-value from the studentized range distribution. The
+        ``(i, j)`` index is the p-value for the ith - jth group comparison.
+
+    Methods
+    -------
+    confidence_inteval() :
```

Comment:
```suggestion
    confidence_interval() :
```
I think that this should accept the `confidence_level` argument, not the `tukeykramer` test itself.

Reply:
I thought about that; would it be a discrepancy that the p-value was generated with the alpha value and then the CI came from a different value?

Reply:
`sig_level` is only used in the confidence interval calculation.